src/main/java/ca/corefacility/bioinformatics/irida/web/controller/api/RESTUsersController.java

```diff
@@ -142,4 +206,11 @@ public ModelMap getCurrentUser() {
 		// get the user from the database.
 		return getResource(u.getId());
 	}
+
+	// TODO: revisit these classes that define the response schemas for openapi
+
+	private class ProjectsSchema {
+		public ResourceCollection<Project> resource;
+	}
```

Comment:
I think this was the only response that returns an object called "projectResources" instead of "resource". If we change the response to return an object called "resource", I can use ResponseResource.java like the other responses. I have removed the TODO.